### PR TITLE
Respect line breaks

### PR DESCRIPTION
**Pull request by nathf**
- Stripping unnecessary whitespace
- Support for custom classNames to field macro

### CODE REVIEW

File: toolkit/templates/summary-table.html [hunk 1]
```diff
@@ -279,7 +279,7 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
 
 
 {% macro multiquestion(value, assurance) -%}
-  {% call field() -%}
+  {% call field(False, False, False, False, False, False, False, '', False, 'nowrap') -%}
```
Comment (sarneaud) [inline toolkit/templates/summary-table.html]: You should be able to do this instead:
{% call field(className='nowrap') %}

Also, the argument isn't _the_ class name; it's an optional extra class.  Maybe extra_classes is a better name.  (Note that the other macro arguments use underscored_names not camelCase.)

**Comment by sarneaud** [comment]
You're also adding a "nowrap" class, which is going to have more effect than stripping whitespace from the HTML, right?

What's the real goal of the PR?  Does it keep the design responsive?

**Comment by nathf** [comment]
There is a corresponding CSS change in the buyer frontend.

The overall goal is if someone enters text into the textarea while creating a brief with line breaks

e.g.

```
Some 

Brief 

Text
```

It is currently rendered as 

```
Some Brief Text
```

This is a precursor PR to support the change in buyer frontend to respect the line breaks

File: toolkit/templates/summary-table.html (hunk 1 above)
Comment (nathf) [reply toolkit/templates/summary-table.html]: Thanks for that catch, updated

**Comment by sarneaud** [comment]
Okay, that helps explain things.  Thanks.

It sounds like you're planning to use one of the "white-space: pre-*" variants, though.  Is that really meant to be called "nowrap"?

**Comment by nathf** [comment]
The corresponding CSS change (in `buyer-frontend`) is:

``` css
.summary-item-field {
    span {
      white-space: pre-wrap;

      * {
        white-space: nowrap;
      }
    }

    &.nowrap {
      span {
        white-space: nowrap;
      }
    }

  }
```

Since not all `summary-item-field`'s are free for all text fields, some required different treatment as they have a predetermined HTML structure

**Comment by sarneaud** [comment]
Wouldn't it be simpler to add a new class for preformatted text, and add that class to the elements that need it?  That would be the text macro on line 172, wouldn't it?

Also, I'm still not understanding why this is nowrap.  The default value is normal.  Is there a reason you don't want the other fields to wrap?

**Pull request merged**